sphinx/tutorial/rose/suites.rst

```diff
@@ -539,22 +549,22 @@ Otherwise an add-hoc web-server can be set up using the
          [env]
          INTERVAL=60
          N_FORECASTS=5
+         WEIGHTING=1
          MAP_TEMPLATE=map-template.html
          SPLINE_LEVEL=0
          WIND_FILE_TEMPLATE=$CYLC_SUITE_WORK_DIR/{cycle}/consolidate_observations/wind_{xy}.csv
          WIND_CYCLES=0, -3, -6
          RAINFALL_FILE=$CYLC_SUITE_WORK_DIR/$CYLC_TASK_CYCLE_POINT/get_rainfall/rainfall.csv
-         # Create the html map file in the task's log directory.
          MAP_FILE=${CYLC_TASK_LOG_ROOT}-map.html
 
-      Finally we need to change the ``forecast`` task to run
-      :ref:`command-rose-task-run`. Change the ``script`` setting, the runtime
-      section for the ``forecast`` task should now look like this:
+      Finally we need to change the ``forecast`` task to instead simply run
```

Comment:
~instead simply~

Reply:
Yes, this was not a very good suggestion, my apologies. Will revert.